LICENSE-APACHE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017-NOW Nikolay Kim
+   Copyright 2017-NOW Actix Team
```

Comment:
Should probably read `Nikolay Kim + Actix Team` unless Nikolay has given the OK for this?

Reply:
I've messaged+emailed him, waiting on a reply. If no response comes back soon I'll revert it.

Reply:
actix team is totally fine, i missed it.